### DISCUSSION TITLE
¿Saben cómo puedo obtener mi correo de Innovación?

### DISCUSSION THREAD

**Discussion by HectorMiguel71** [LaunchX-InnovaccionVirtual/CursoIntroPython#312] [General]
¿Saben cómo puedo obtener mi correo de Innovación y poder enviar mis katas?

**Comment by HectorMiguel71** [comment]
@FernandaOchoa ¿Sabrás cómo? Ya terminé los ejercicios pero no logro acceder al curso de FRONT ¿podrás ayudarme por favor?

**Reply by YomairaBarrera** [reply]
Hola compañero te dejo el link en donde están las instrucciones para el llenado del forms y entrega de katas.
https://github.com/LaunchX-InnovaccionVirtual/CursoIntroPython/discussions/232

- Tu correo de innovaccion es la cuenta con la que inicias sesión en el Teams de Microsoft para poder revisarlo tienes que abrir ese correo en Outlook, ya sea en la aplicación de desktop o inclusive en el link que te puse anteriormente esta la pagina web igual.

**Reply by HectorMiguel71** [reply]
Muchas gracias @YomairaBarrera el detalle es que no tengo correo de innovación sólo me llegó esta invitación pero ningún acceso extra
![image](https://user-images.githubusercontent.com/99772437/154810089-ac7ec9e0-4bfd-4002-95a5-a37048745a45.png)

**Reply by YomairaBarrera** [reply]
manda mensaje al insta de innovaccion virtual, pero como le hiciste para entrar al teams si no tienes tu cuenta de Launch X?

**Reply by gersondlarosa** [reply]
Igual en el instagram de inovaccion te responden en el chat
https://www.instagram.com/innovaccionvirtual/?hl=es-la

**Reply by HectorMiguel71** [reply]
@YomairaBarrera  @gersondlarosa  efectivamente así fue por instagram que me resolvieron. Gracias por su ayuda
